dataset.py

```diff
@@ -128,6 +128,20 @@ def pull_item(self, index):
 
         # 2. build list of annotation
         anno_list = []
+        if not dataset_item.attributes['detection']:
+            # NOTE: add annotation as background ( no object )
+            # label_id: -1 will be incremented and used for background
+            dataset_item.attributes['detection'] = [
+                {
+                    'label_id': -1,
+                    'rect': {
+                        'xmin': width,
+                        'ymin': height,
```

Comment:
I am sorry I'm not sure but I think `xmin` and `ymin` must be `0` ...?

Reply:
It's better to set (0, 0, 0, 0) ,or  I saw some implemetation use  (-1, -1, -1, -1) to find the default values.
It depends on how to recognize it.

Reply:
thanks, I will try with (0, 0, 0, 0) and see if it works well.

Reply:
I've tested with (0, 0, 0, 0) and checked if this works :)